covid19dp_submission/download_analyses.py

```diff
@@ -104,11 +94,15 @@ def get_analyses_to_process(project, num_analyses, total_analyses, processed_ana
 
 
 @retry(logger=logger, tries=4, delay=120, backoff=1.2, jitter=(1, 3))
-def get_analyses_from_ena(project, offset, limit):
+def get_analyses_from_ena(project, offset=0, limit=0):
```

Comment:
Maybe add a comment somewhere explaining why we're doing this (just in case they ever lift the restriction...)

Reply:
Good point. I've added a comment.